### PR TITLE
Support default framebuffers besides 0

### PR DESCRIPTION
**Pull request by loganmc10**
mupen64plus now includes a "CoreVideo_GL_GetDefaultFramebuffer" API function that allows the GFX plugin to query what the default framebuffer is. Some platforms (iOS, RetroArch) use a non-0 default framebuffer.

@JoeMatt @braindx can review/test this, they maintain Provenance, which is an iOS app that includes mupen64plus+GLideN64

### CODE REVIEW

File: src/Graphics/OpenGLContext/windows/windows_DisplayWindow.cpp [hunk 1]
```diff
@@ -298,3 +299,8 @@ void DisplayWindowWindows::_readScreen(void **_pDest, long *_pWidth, long *_pHei
 	glReadPixels(0, m_heightOffset, m_width, m_height, GL_RGB, GL_UNSIGNED_BYTE, *_pDest);
 #endif
 }
+
+graphics::ObjectHandle DisplayWindowWindows::getDefaultFramebuffer()
```
Comment (gonetz) [inline src/Graphics/OpenGLContext/windows/windows_DisplayWindow.cpp]: _getDefaultFramebuffer, no?

**Comment by gonetz** [comment]
@JoeMatt @braindx please review.

File: src/Graphics/OpenGLContext/windows/windows_DisplayWindow.cpp (hunk 1 above)
Comment (loganmc10) [reply src/Graphics/OpenGLContext/windows/windows_DisplayWindow.cpp]: Ah yes thanks, I can't compile for Zilmar spec so I did that part blind

**Comment by JoeMatt** [comment]
Sure, I'll take a look this weekend.

**Pull request merged**